src/datasources/cache/cache.service.interface.ts

```diff
@@ -12,4 +12,9 @@ export interface ICacheService {
   get(cacheDir: CacheDir): Promise<string | undefined>;
 
   deleteByKey(key: string): Promise<number>;
+
+  incrementAndGet(
```

Comment:
I think `incrementAndGet` is more descriptive for the clients of this interface API than simply `incr`, but of course suggestions on the naming are welcome.

Reply:
I would rename this to just `increment`. The result of the increment can be documented instead. Else it seems that we are composing different actions which should be the responsibility of the caller.

Example: the caller might choose to ignore the contents of `increment` which makes `increment` self-explanatory. But the result would still be there if needed.

Reply:
Done in [66f1152](https://github.com/safe-global/safe-client-gateway/pull/1225/commits/66f1152d921315ee265df6e48f67afcb63edead5)